test/support/topology_test_data.ex

```diff
@@ -0,0 +1,194 @@
+defmodule Mongo.TopologyTestData do
+  def single, do: %{
+    set_name: nil,
+    type: :single,
+    compatibility_error: nil,
+    compatible: true,
+    local_threshold_ms: 15,
+    max_election_id: nil,
+    max_set_version: nil,
+    servers: %{
+      "localhost:27017" => %{
+        address: "localhost:27017",
+        arbiters: [],
+        election_id: nil,
+        error: nil,
+        hosts: [],
+        last_update_time: nil,
+        last_write_date: nil,
+        max_wire_version: 4,
+        me: nil,
+        min_wire_version: 0,
+        op_time: nil,
+        passives: [],
+        primary: nil,
+        round_trip_time: 44,
+        set_name: nil,
+        set_version: nil,
+        tag_set: %{},
+        type: :standalone
+      }
+    }
+  }
+
+  def repl_set_with_master, do: %{
+    compatibility_error: nil,
+    compatible: true,
+    local_threshold_ms: 15,
+    set_name: "replset1",
+    type: :replica_set_with_primary,
+    max_election_id: nil,
+    max_set_version: 3,
+    servers: %{
+      "cjmbp.local:27018" => %{
```

Comment:
These names will work for everyone as this data is just used for tests that do not access the database. With that said, we probably should change it to `localhost`, or some other domain for the sake of consistency.